packages/devtools_app/lib/src/debugger/debugger_controller.dart

```diff
@@ -89,6 +89,9 @@ class DebuggerController extends DisposableController
 
   VmServiceWrapper get _service => serviceManager.service;
 
+  Map<ObjRef, Future<Set<String>>> autocompleteCache = {};
+  Map<ObjRef, Future<Set<String>>> autocompleteInclusiveCache = {};
```

Comment:
can you add dart doc to these to explain the difference?

Reply:
done. tried tweaking the names as well to make them a bit clearer.

Reply:
+1 - it's not immediately obvious to me